web/static/scss/_components.scss

```diff
@@ -115,6 +115,10 @@
   a.btn {
     margin: 0.7em 0 3em 0;
   }
+
+  form {
```

Comment:
What is this changing visually? Generally not preferred to use html tags for CSS